server/src/main/java/org/bonitasoft/console/common/server/utils/ContractTypeConverter.java

```diff
@@ -72,6 +73,11 @@ public Object convertToType(final Type type, final Serializable parameterValue)
         return convertToType(clazz, parameterValue);
     }
 
+    public Map<String,Serializable> getProcessedInput(ContractDefinition processContract, Map<String, Serializable> inputs, long maxSizeForTenant, long tenantId, boolean deleteFile) throws FileNotFoundException {
+        this.deleteFile = deleteFile;
+        return this.getProcessedInput(processContract, inputs, maxSizeForTenant, tenantId);
```

Comment:
I think you should have done the opposite: have the former method call the new one to avoid to have a new object attribute to manage...